server/src/main/java/org/elasticsearch/cluster/ClusterState.java

```diff
@@ -474,6 +497,7 @@ public enum Metric {
         BLOCKS("blocks"),
         NODES("nodes"),
         METADATA("metadata"),
+        TRANSPORT_VERSIONS("transport_versions"),
```

Comment:
IMO there's no need to introduce a new `Metric` for this - I don't see a good reason for clients to need to request it separately from nodes so I'd suggest just using `NODES` to cover both.

Reply:
OK. Is it worth merging the nodes & versions info together into the same JSON structure in the xcontent write, or just leave it a separate field that is triggered on NODES?

Reply:
Two fields is fine with me.